src/pages/Dashboard.tsx

```diff
@@ -61,6 +61,7 @@ const Dashboard = () => {
         });
 
         const data = await resp.json();
+        data.sort((a: any, b:any) => a.name.localeCompare(b.name));
```

Comment:
Sorry, completely missed the typings! Instead of allowing `any` type, could you please change it to only allow a string for the `a` and `b` parameters?

Reply:
For sure! I changed the type to `IStepProps`.